lib/ar2dto/dto.rb

```diff
@@ -13,7 +13,7 @@ def [](original_model)
     end
 
     def self.inherited(base)
-      base.include ::AR2DTO::ActiveModel
+      base.include ::AR2DTO::ActiveModel if ::AR2DTO::Config.instance.active_model_compliance
```

Comment:
Just to check, this right now is a global config, and is intended to remain like that, or do you plan to also add it at class level too?